src/main/resources/config/summary/COMP.json

```diff
@@ -17,12 +17,17 @@
           "value": "Ex-Gratia"
         },
         {
-          "label": "Minor Misconduct",
```

Comment:
Should these be lower case - they feel like a proper noun to me? Resolve this commend if UCD have requested the lowercase 🙂

Reply:
Sentence case has been requested here